raiden-dapp/src/store/index.ts

```diff
@@ -180,6 +180,9 @@ const store: StoreOptions<RootState> = {
     network: (state: RootState) => {
       return state.network.name || `Chain ${state.network.chainId}`;
```

Comment:
Notice `network.name` is always defined in ether's `Network`. With unknown network, it's simply `unknown`, that's why we have [this approach](https://github.com/raiden-network/light-client/blob/6f65f2d6ba0221a8ecc774e207c6782e71c1ab02/raiden-ts/src/utils/ethers.ts#L164) on SDK, which you can use or copy, as it's simple enough.

Reply:
It would make sense to replace it with the `getNetwork` function instead of duplicating the code.